pkg/info/info.go

```diff
@@ -0,0 +1,55 @@
+package info
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+	"unicode"
+
+	"github.com/weaveworks/eksctl/pkg/version"
+)
+
+//go:generate go run ./release_generate.go
+
+// Version holds versions info
+type Version struct {
+	EksctlVersion  string
+	KubectlVersion string
+	OS             string
+}
+
+// GetVersion returns versions info
+func GetVersion() Version {
+	return Version{
+		EksctlVersion:  version.String(),
+		KubectlVersion: getKubectlVersion(),
+		OS:             runtime.GOOS,
+	}
+}
+
+// getKubectlVersion returns the kubectl version
+func getKubectlVersion() string {
+	cmd := exec.Command("kubectl", "version", "--short", "--client")
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		return fmt.Sprintf("error : %v", err)
+	}
+
+	values := strings.FieldsFunc(string(out), func(c rune) bool {
+		return unicode.IsSpace(c)
+	})
+	if len(values) == 0 {
+		return "unknown version"
+	}
+	return values[len(values)-1]
+}
+
+// String returns eksctl info as a string
+func String() string {
+	if data, err := json.Marshal(GetVersion()); err == nil {
+		return string(data)
+	}
+	return ""
```

Comment:
Can we do the inverse of this?
```suggestion
	if data, err := json.Marshal(GetVersion()); err != nil {
		return fmt.Sprintf("error : %v", err)
	}
	return string(data)
```